gammapy/data/tests/test_observation_stats.py

```diff
@@ -0,0 +1,71 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from numpy.testing import assert_allclose
+from astropy.tests.helper import pytest
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from ...data import DataStore, ObservationList, ObservationStats, Target
+from ...utils.testing import requires_data
+from ...extern.regions.shapes import CircleSkyRegion
+from ...background import reflected_regions_background_estimate as refl
+from ...image import ExclusionMask
+
+
+@requires_data('gammapy-extra')
+def get_obs_list():
+    data_store = DataStore.from_dir(
+        '$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
+    run_list = [23523, 23526]
+    obs_list = ObservationList([data_store.obs(_) for _ in run_list])
+    return obs_list
+
+
+@requires_data('gammapy-extra')
+def get_obs(id):
+    obs_list = get_obs_list()
+    for run in obs_list:
+        if run.obs_id == id:
+            return run
+
+
+def get_target():
```

Comment:
Making this a pytest fixture would be a bit better.
https://github.com/jiffyclub/pytest-features#fixtures

I.e. put this here:
```python
@pytest.fixture
def target():
    ... same as what you have now ...
```

and then inject `target` objects into the tests that use it, i.e. below:
```python
def test_str(target):
```

At the moment we're not using fixtures a lot in Gammapy tests, but this is something that we'll gradually clean up ... almost always input test data will be injected into tests via fixtures.